examples/fabric/byot/run.py

```diff
@@ -0,0 +1,80 @@
+import torch
```

Comment:
People won't know what BYOT is. I suggest renaming the directory to `build_your_own_trainer`